react-native/react/menubar/index.render.desktop.js

```diff
@@ -9,10 +9,12 @@ import {intersperse} from '../util/arrays'
 import {parseFolderNameToUsers, canonicalizeUsernames, stripPublicTag} from '../util/kbfs'
 
 import {globalStyles, globalColors} from '../styles/style-guide'
-import {Text, Input} from '../common-adapters/index.desktop.js'
+import {Text, Input, Terminal, Icon} from '../common-adapters/index.desktop.js'
 
 import {CircularProgress} from 'material-ui'
-import {cleanup} from '../util/kbfs'
+import {cleanup, allowLoggedOut as allowLoggedOutKBFS} from '../util/kbfs'
+// TODO use this instead of notification after merging
+// import {NotifyPopup} from '../native/notifications'
```

Comment:
It looks like that exists in this branch, no? (https://github.com/keybase/client/blob/nojima/DESKTOP-291-menu-loggedout/react-native/react/native/notifications.desktop.js)